usage_examples/rel_openai/README.md

```diff
@@ -0,0 +1,38 @@
+# Relation extraction using LLMs
+
+This example shows how you can use a model from OpenAI for relation extraction in
+zero- or few-shot settings.
+
+Here, we use the pretrained [`en_core_web_sm` model](https://spacy.io/models/en#en_core_web_sm)
```

Comment:
At this point I think we are not recommending using the `en_core_web_sm` for anyone, the `en_core_web_lg` I think should be efficient enough for most people at this point?

Reply:
I would strongly recommend using `en_core_web_md` instead of `en_core_web_lg` for things like this. Way smaller, performance nearly the same.

Reply:
I changed the config to `en_core_web_md`